torch/_inductor/autotune_process.py

```diff
@@ -186,7 +214,8 @@ def benchmark(
         mod = PyCodeCache.load_by_key_path(self.module_cache_key, self.module_path)
         if DEBUG:
             print(
-                f"benchmark module key: {self.module_cache_key}, path: {self.module_path}"
+                f"benchmark module key: {self.module_cache_key}, path: {self.module_path}",
+                file=sys.stderr,
```

Comment:
it seems a bit off to log this to stderr. is this to avoid parsing it back on the main thread ?

Reply:
@eellison, yeah, see my comment above to skylion007. These print stmts were already going nowhere and I just changed them to stderr in case anyone ever enabled DEBUG above (but they'll still go nowhere). Accidentally writing to stdout would mess up the message passing between processes and is pretty confusing. If we want to clean up the logging in this diff, I'd suggest either 1) having the sub-process log to a tmp file somewhere, or 2) We can continue to log to stderr (but probably using a real logger), read the stderr from the pipe in the main process, and log it on the sub-process's behalf. What do you think?

Reply:
I'll go the route of logging to a file